tests/core/channels/test_twilio_voice.py

```diff
@@ -0,0 +1,55 @@
+import json
+import logging
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_twilio_voice_twiml_response_text():
+
+    from rasa.core.channels.twilio_voice import TwilioVoiceInput
+    from rasa.core.channels.twilio_voice import TwilioVoiceCollectingOutputChannel
+
+    tv = TwilioVoiceInput("woman")
+
+    output_channel = TwilioVoiceCollectingOutputChannel()
+
+    await output_channel.send_text_message(recipient_id="Chuck Norris", text="Test:")
```

Comment:
how about also adding a test for multiple responses and pauses?

Reply:
Hmmm seems like there would need to be a model for this to process the text inputs to the webhook. Do we have an example of a channel connector test that's connected to a model?

Reply:
You could maybe use something like [here](https://github.com/RasaHQ/rasa/blob/13696886922f2d09ad47a9c1a5cb846573bf91bf/tests/core/channels/test_hangouts.py#L26). 

not sure why you need a model though. Couldn't you just run `output_channel.send_text_message` twice?